packages/neuron-wallet/src/controllers/app/menu.ts

```diff
@@ -317,6 +317,8 @@ const updateApplicationMenu = (mainWindow: BrowserWindow | null) => {
             t(`messageBox.multisig-address.title`),
             {
               width: 900,
+              maxWidth: 900,
+              minWidth: 900,
               resizable: true
```

Comment:
Why not set `resizable` false?

Reply:
> Why not set `resizable` false?

With this set, the window is allowed to resize vertically to add more `input fields` but not allowed to resize horizontally.